src/views/Deposit/components/index.ts

```diff
@@ -1,78 +1,79 @@
 import styled, { css } from 'styled-components';
 import { media } from 'components/breakpoints';
-import { colors } from 'components/theme';
+import { colors, GU } from 'components/theme';
 
 export { EnterAmount } from './EnterAmount';
 export { InfoCard } from './InfoCard';
 
 interface IStyledButtonLarge {
-    active: boolean;
+  active: boolean;
 }
 
 export const StyledButtonLarge = styled.button<IStyledButtonLarge>`
-    background: transparent;
-    border: none;
-    font-family: PixelSplitter;
-    font-size: 8px;
-    height: 40px;
-    letter-spacing: 3px;
-    outline: none;
-    position: relative;
-    transition: all .3s ease;
-    width: 50%;
+  background: transparent;
+  border: none;
+  font-family: PixelSplitter;
+  font-size: 0.8rem;
+  height: ${10 * GU}px;
+  letter-spacing: 3px;
+  outline: none;
```

Comment:
Let's *not* do this. Buttons need to be accessible by default and this is, therefore, a bad practice. If the focus style doesn't, let's ask Valeria to create a focus state that is pretty and implement it manually with the `:focus` selector :)

Reply:
Fixed in c09a156687bc5bea9c84f73335c0c6246df7dfcb